src/platform/cordova.ios.scss

```diff
@@ -3,9 +3,13 @@
 // iOS Cordova
 // --------------------------------------------------
 
+/// @prop - Height of the Statusbar
+$cordova-ios-statusbar-padding:                         20px   !default;
```

Comment:
LGTM!
just one thing, not sure if it was done on purpose: there are 3 spaces between 20px and !default